docs/usermanual-fonts-and-faces.xml

```diff
@@ -5,20 +5,449 @@
   <!ENTITY version SYSTEM "version.xml">
 ]>
 <chapter id="fonts-and-faces">
-  <title>Fonts and faces</title>
-  <section id="using-freetype">
-    <title>Using FreeType</title>
+  <title>Fonts, faces, and output</title>
+    <para>
+      In the previous chapter, we saw how to set up a buffer and fill
+      it with text as Unicode code points. In order to shape this
+      buffer text with HarfBuzz, you will need also need a font
+      object.
+    </para>
+    <para>
+      HarfBuzz provides abstractions to help you cache and reuse the
+      heavier parts of working with binary fonts, so we will look at
+      how to do that. We will also look at how to work with the
+      FreeType font-rendering library and at how you can customize
+      HarfBuzz to work with other libraries.
+    </para>
+    <para>
+      Finally, we will look at how to work with OpenType variable
+      fonts, the latest update to the OpenType font format, and at
+      some other recent additions to OpenType.
+    </para>
+
+  <section id="fonts-and-faces-objects">
+    <title>Font and face objects</title>
+    <para>
+      The outcome of shaping a run of text depends on the contents of
+      a specific font file (such as the substitutions and positioning
+      moves in the 'GSUB' and 'GPOS' tables), so HarfBuzz makes
```

Comment:
The user manual is inconsistent about how to style tags: `abcd`, `'abcd'`, `"abcd"`, or `<literal>abcd</literal>`?

Reply:
My personal belief is that table & feature names match <token> best (literal is too generic; it's what I've used when I can't find a good match elsewhere in docbook); the docbook <term> kind of just doesn't match specification requirements, since it's so code-focused. But that's something that I need to do in a full pass on all the docs. The a-priori material was also inconsistent, and in the interest of time I have not been trying to rectify those while also getting the new text written. Results will be better when markup-consistency is its own pass, IMO.